### DISCUSSION TITLE
Problem with Focusrite 2i2

### DISCUSSION THREAD

**Discussion by corrados** [jamulussoftware/jamulus#912] [Technical Support & Help]
I just received the following question and I will use that to try out this new discussion feature :-)

Here is the question. If you can help, please post your support:

**I have a Focusrite 2i2 (3rd generation) Studio (with a condenser mic).
When I log in to a Jamulus server, it works fine with my Input 2- but not my Input 1.
I am using the newest Foscurite ASIO.
When I plug my microphone into Input 1, I can see the input (greenlights) on the "Input" section of the Jamulus client, BUT those green lights do NOT show up on the mixer side - and nobody in the Jamulus meeting can hear me.
When I switch to Input 2, the green lights on the mixer side show up and everyone can hear me.
Any idea how Jamulus can be recognizing Input 2 but NOT Input 1?
By the way: when I use the Focusrite with any other software, both inputs are recognized just fine.
ALSO when I plug the Focusrite in to a different computer, and use Jamulus in the same way, BOTH inputs are recognized.
So it seems clear that there is something in my own laptop set-up that is disabling Input 1 - but I can't figure out what it would be.
(I tried uninstalling every other program I could think of that might be "grabbing" that input - Ableton Live, ASIO4All, Melodyne, Reaper, Reaplugs, Obs audio, VBCable, Voicemeeter, etc.) - I uninstalled them all and tried again -but no change.
Any ideas you have for why this isn't working with my laptop would be most appreciated!**

**Comment by pljones** [comment]
OK, I can reproduce this.

Set up a Reaper track with a Stereo pair hardware input where only the "left" (first) input is connected.
ReaRoute that pair into Jamulus.
Set up Jamulus with Mono-in/Stereo Out.
Set (local) Pan to L-50
Connect to a server
You can hear yourself
Set (local) Pan to R-50
You cannot hear yourself
Switch the mono hardware input from "left" to "right" (second) input
You can hear yourself

**Reply by pljones** [reply]
My guess is the person pulled both input sliders (pan and reverb) to the bottom, rather than centring pan, at some point.

**Reply by reinhardwh** [reply]
> My guess is the person pulled both input sliders (pan and reverb) to the bottom, rather than centring pan, at some point.

Confirmed, this happened to me as well. Setting slider correctly will resolve the issue.
